cellseg_gsontools/main.py

```diff
@@ -0,0 +1,26 @@
+from base import gauss2d
```

Comment:
`main.py` files don't need to be pushed to the repo. If you want to do that, put them in a `scripts` folder for example and give them descriptive filenames